widget/infprogressbar_test.go

```diff
@@ -0,0 +1,54 @@
+package widget
+
+import (
+	"testing"
+	"time"
+
+	"fyne.io/fyne"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInfiniteProgressBar_Creation(t *testing.T) {
+	bar := NewInfiniteProgressBar()
+	// ticker should be nil when created
+	assert.Nil(t, bar.ticker)
+}
+
+func TestInfiniteProgressBar_Ticker(t *testing.T) {
+	bar := NewInfiniteProgressBar()
+
+	bar.Show()
```

Comment:
I don't think we can rely on users calling bar.Show() - this is not the semantic of other widgets as they are visible by default...

Reply:
I removed the `Show()` call from the tests by calling `Start()` in `NewProgressBarInfinite()`.  Same effect.